src/Cabal2Nix/PostProcess.hs

```diff
@@ -72,7 +72,8 @@ postProcess' deriv@(MkDerivation {..})
   | pname == "leksah-server"    = deriv { buildDepends = Set.insert "process-leksah" buildDepends }
   | pname == "lhs2tex"          = deriv { extraLibs = Set.insert "texLive" extraLibs, phaseOverrides = lhs2texPostInstall }
   | pname == "libffi"           = deriv { extraLibs = Set.delete "ffi" extraLibs }
-  | pname == "liquid-fixpoint"  = deriv { buildTools = Set.insert "ocaml" buildTools }
+  | pname == "liquid-fixpoint"  = deriv { buildTools = Set.insert "ocaml" buildTools, configureFlags = Set.insert "-fbuild-external" configureFlags }
+  | pname == "liquidhaskell"    = deriv { doCheck = False } -- test-suite requires cvc4 or z3
```

Comment:
That's a pretty substantial buildInput and I'd rather let users choose which SMT solver they want to use. Is it possible to add a buildInput that will only be realized if `doCheck` is enabled?